giraffe/webpack.config.js

```diff
@@ -3,10 +3,12 @@ const webpack = require('webpack')
 const pkg = require('./package.json')
 
 module.exports = {
-  entry: './src/index.ts',
+  entry: {
```

Comment:
This comment applies to all the other questions in this PR.

As a Giraffe user I would be slightly concerned that there is a lot of widgets included but I will use only a few of them on a single page. This dead weight will cause increased load times of the page. This has impact on user experience, google penalizations etc. 

One solution to overcome this issue is to load the code for individual widgets on demand. This would be effective in the case of the geo widget since it is using quite a few external libraries. Right now half the code of giraffe is geo and its libraries and I think that the load time is significant - in my experience it may add even seconds to the load time. 

So I tried to do some investigation to load geo on demand. I was able to build giraffe in a way that there were two separate chunks/entry points but I struggled to incorporate it into an external application in an easy/transparent way. It may even be framework dependant. It didn't have time to resolve it, I plan to return to this once I will be embedding geo into chronograf.